src/libopensc/cards.h

```diff
@@ -22,6 +22,7 @@
 #define _OPENSC_CARDS_H
 
 #include "libopensc/types.h"
```

Comment:
clang-format suggested the following change:


```
-#include "libopensc/types.h"
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/cards.h -->